### PR TITLE
OFMCC-4741 - Header updates

### PR DESCRIPTION
**Pull request by weskubo-cgi**
- Add help and home icons to header
 - Update layout for responsiveness
 - Clean up styles
 - Rename "Resources" to "Help"

### CODE REVIEW

File: frontend/src/views/HomeView.vue [hunk 1]
```diff
@@ -45,7 +45,7 @@
         </v-card>
       </v-col>
       <v-col cols="12" md="6" lg="4">
-        <v-card class="home-card" id="resources-card" prepend-icon="mdi-help" title="Resources" @click="$router.push({ name: 'resources' })">
+        <v-card class="home-card" id="help-card" prepend-icon="mdi-help" title="Resources" @click="$router.push({ name: 'help' })">
```
Comment (vietle-cgi) [inline frontend/src/views/HomeView.vue:48]: should we update its title to Help?

File: frontend/src/components/TheHeader.vue [hunk 2]
```diff
@@ -65,15 +78,14 @@ export default {
...
-    appTitleXs() {
-      const index = this.appTitle.indexOf(' ')
-      return `${this.appTitle.substring(0, index)}<br/>${this.appTitle.substring(index + 1)}`
-    },
-    xl() {
-      return this.$vuetify.display.xl
-    },
-    xs() {
-      return this.$vuetify.display.xs
+    xs() {},
```
Comment (vietle-cgi) [inline frontend/src/components/TheHeader.vue:81]: we can remove this empty xs()

**Review by vietle-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/views/HomeView.vue (hunk 1 above)
Comment (weskubo-cgi) [reply frontend/src/views/HomeView.vue]: Done.

File: frontend/src/components/TheHeader.vue (hunk 2 above)
Comment (weskubo-cgi) [reply frontend/src/components/TheHeader.vue]: Removed.

**Review by vietle-cgi — APPROVED** [review]

**Pull request merged**